doc/12-icinga2-api.md

```diff
@@ -890,9 +890,15 @@ curl -k -s -S -i -u root:icinga -H 'Accept: application/json' \
 Existing objects must be modified by sending a `POST` request. The following
 parameters need to be passed inside the JSON body:
 
-  Parameters | Type       | Description
-  -----------|------------|---------------------------
-  attrs      | Dictionary | **Required.** Set specific object attributes for this [object type](09-object-types.md#object-types).
+| Parameters     | Type       | Description                                                                                                                |
+|----------------|------------|----------------------------------------------------------------------------------------------------------------------------|
+| attrs          | Dictionary | **Optional.** Set specific object attributes for this [object type](09-object-types.md#object-types).                      |
+| restore\_attrs | Array      | **Optional.** Discard modifications of specific object attributes for this [object type](09-object-types.md#object-types). |
+
+!!! info
+
+  If a particular attribute is given in both sets,
+  it's first restored and then set to the desired new value.
```

Comment:
I'm not sure if this will render correctly with mkdocs. The other info blocks use 4 spaces for indentation which makes them a code block in pure markdown (which this isn't).